SendWithUs.Client/Requests/SendRequestConverter.cs

```diff
@@ -128,21 +141,21 @@ protected void WriteRecipientsList(JsonWriter writer, JsonSerializer serializer,
             foreach (var address in recipients)

```

Comment:
Not a big deal, but find it weird that you call the loop variable `address` when it's coming from something called `recipients` (the noun you chose for your domain) and we're translating it to an `address` in the SendWithUs domain.